src/IniFile.h

```diff
@@ -42,14 +42,14 @@ class cIniFile : public cSettingsRepositoryInterface
 
 	struct key
 	{
-		std::vector<AString> names;
-		std::vector<AString> values;
-		std::vector<AString> comments;
+		std::vector<AString> m_names;
+		std::vector<AString> m_values;
+		std::vector<AString> m_comments;
```

Comment:
If you're changing these anyway then shouldn't it be `m_Names`, `m_Values`, etc.

Reply:
Should be the `readability-identifier-naming.PrivateMemberCase` check. With the value of `CamelCase`

Reply:
@bibo38 I'm currently avoiding adding new checks since every new check adds 300+ new changes that I need to make. There are a lot of checks that need to be added when the basics work.